extensions-builtin/Lora/lora.py

```diff
@@ -138,6 +138,50 @@ def __init__(self):
         self.down = None
         self.alpha = None
 
+class LoRaCacheLRU:

```

Comment:
Could this just be implemented with `functools.lru_cache`?

Reply:
I didn't know about this before. If I use `functools.lru_cache`, it's quite simple, right? I just need to add the decorator before the `load_lora` function, right?